contracts/RewardsSender.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "./Bridge.sol";
```

Comment:
https://docs.soliditylang.org/en/v0.8.10/types.html#members-of-addresses

Take a look at the `call` function.